CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Added pylint `line-too-long` and `invalid-name` ([#590](https://github.com/opensearch-project/opensearch-py/pull/590))
 - Added pylint `pointless-statement` ([#611](https://github.com/opensearch-project/opensearch-py/pull/611))
 - Added a log collection guide ([#579](https://github.com/opensearch-project/opensearch-py/pull/579))
+- - Update the GHA release workflow with trusted publisher enabled  ([#614](https://github.com/opensearch-project/opensearch-py/pull/614))
```

Comment:
I would say "Added GHA release"

Reply:
Updated here. Please help review it again. Thanks.